docs/docs/spec.md

```diff
@@ -1,4 +1,2 @@
 <iframe width="1400" height="600" src="../spec/openapi.html"></iframe>
 [full window :octicons-link-external-16:](../spec/openapi.html)
-
-[Here](./spec/openapi.json) is the OpenAPI spec.
```

Comment:
I didn't expect the link to come out; just to point to `../spec/openapi.json` instead. The JSON file gets generated by the deploy process, so the file won't be there until it's deployed.